source/features/edit-comments-faster.tsx

```diff
@@ -14,12 +14,12 @@ function init(): void {
 
 			comment
 				.closest('.js-comment')!
-				.querySelector('.timeline-comment-actions > details:last-child')! // The dropdown
+				.querySelector('.timeline-comment-actions > details:last-child, .timeline-comment-actions details:last-child')! // The dropdown
```

Comment:
Did you check that this does not make issues in other places. There is probably a reason that we used such a specific selector.

Reply:
See https://github.com/sindresorhus/refined-github/pull/4042#discussion_r584462656

Reply:
I only tested in issues, Discussions and this PR page (all non-locked).
I don't have any locked conversation URL to have participated in.

As I wrote in the OP, I can only suggest a more specific alternative selector:
`.timeline-comment-actions > div > details:last-child`

Reply:
Either way the 2 selectors currently overlap. The first one is unnecessary because the second one already covers that. 

@yakov116’s notes still apply

Reply:
Ok, I kept only the second one. Thank you.

Reply:
Did you check if locked issues work?

Reply:
Yes, I tried it in a test repo I have made:
I locked an issue, discussion and PR page, and it works ok in all 
with the selector I kept `.timeline-comment-actions details:last-child`.